luarules/gadgets/unit_inherit_creation_xp.lua

```diff
@@ -82,70 +79,69 @@ local lastRunFrame = 0
 local oldChildXPValues = {}
 function gadget:GameFrame(frame)
 	if frame > (lastRunFrame + 30) then
```

Comment:
I recommend using the modulo operator. Try using incongruent numbers across the board to spread load.

Reply:
... Gonna Google that right quick...